script/ddgc_generate_traffic.pl

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env perl
+
+use strict;
+use warnings;
+
+$ENV{DDGC_IA_AUTOUPDATES} = 0;
+
+use FindBin;
+use lib $FindBin::Dir . "/../lib";
+
+use DDGC;
+use List::MoreUtils qw(zip);
+use List::Util qw(pairs);
+use DateTime;
+
+my $d = DDGC->new;
+
+my @ids = $d->rs('InstantAnswer')
+    ->search({dev_milestone => 'live'})
+    ->get_column('meta_id')
+    ->all;
+
+my $today = DateTime->now();
+my $last_month = $today->clone()->subtract(days => 31);
+
+my @days = map { $last_month->clone->add(days => $_)->strftime('%F') } (1..31);
```

Comment:
Might be worth pulling the number of days out so we can specify how many days of data we want to generate.